PhysicsTools/Utilities/src/LumiReWeighting.cc

```diff
@@ -157,25 +157,12 @@ double LumiReWeighting::weight( float npv ) {
 
 double LumiReWeighting::weight( const edm::EventBase &e ) {
 
-  // find provenance of event objects, just to check at the job beginning if there might be an issue  
-
+  // get pileup summary information
   if(FirstWarning_) {
-
-    const edm::ProcessHistory& PHist = e.processHistory();
-    edm::ProcessHistory::const_iterator PHist_iter = PHist.begin();
-
-    for(; PHist_iter<PHist.end() ;++PHist_iter) {
-      edm::ProcessConfiguration PConf = *(PHist_iter);
-      const edm::ReleaseVersion& Release =  PConf.releaseVersion() ;
-      const std::string& Process =  PConf.processName();
-
-    }
+    e.processHistory(); // keep the function call
```

Comment:
@mrodozov - it seems to me that a removed comment on line 160 has more sense than this one.

Reply:
@ianna - updated